### PR TITLE
postman: initial commit

### PR DESCRIPTION
**Pull request by ikstream**
Signed-off-by: Stefan Venz <stefan.venz@protonmail.com>

### CODE REVIEW

File: packages/postman/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# Original-Maintainer: Jay Ta'ala <jay@jaytaala.com>
+# Original-Contributor: Claudio d'Angelis <claudiodangelis at gmail dot com>
+pkgname=postman
+_pkgname=Postman
+pkgver=9.9.3
+pkgrel=1
+pkgdesc='Build, test, and document your APIs faster'
+provides=('postman')
+conflicts=('postman')
+arch=('x86_64')
```
Comment (noraj) [inline packages/postman/PKGBUILD:14]: lacks a `groups`

**Review by noraj — CHANGES_REQUESTED** [review]
Postman is not a security tool and can already be installed via [AUR](https://aur.archlinux.org/packages/postman-bin).
I don't see the interest of adding it in BA.

**Comment by ikstream** [comment]
> Postman is not a security tool and can already be installed via [AUR](https://aur.archlinux.org/packages/postman-bin). I don't see the interest of adding it in BA.

It is a good tool for Testing APIs, especially if you get Postman Collections from the developers.

**Comment by noptrix** [comment]
@ikstream thank you

**Pull request merged**